performance/src/main/java/uk/gov/gchq/palisade/example/perf/analysis/PerfCollector.java

```diff
@@ -98,16 +98,14 @@ public static PerfStats computePerfStats(final List<Long> results) {
 
         // compute mean and standard deviation
         double total = 0;
-        for (double seconds : resultList) {
-            total += seconds;
-        }
-        stats.setMean(total / stats.getNumTrials());
         // std.dev. requires mean computing first
         double totalDiff = 0;
         for (double seconds : resultList) {
             double difFromMean = seconds - stats.getMean();
+            total += seconds;
             totalDiff += (difFromMean * difFromMean);
         }
+        stats.setMean(total / stats.getNumTrials());
```

Comment:
Revert this.
The first bit calculates the mean (`stats.setMean()`) and the second uses it (`stats.getMean()`)
With these changes, the `.getMean()` will always be `0` where used.

Reply:
Okay, I will revert these changes but this may need a warning as this had the same smell as the above comment.